### PR TITLE
Automox.AutomoxAgent version 1.42.22

### PR DESCRIPTION
**Pull request by s-crypt**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/143315)

### CODE REVIEW

**Comment by s-crypt** [comment]
Well I have been able to visit those URLs on multiple networks and devices, so it was probably a bot block on their side.

**Comment by stephengillie** [comment]
This might be due to a network block of data centers, to prevent automated downloads. The `/hc/` in the URL probably indicates they're using the ZenDesk HelpCenter, which seems to frequently blocks our data centers.

File: manifests/a/Automox/AutomoxAgent/1.42.22/Automox.AutomoxAgent.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+- Agent
+- MDM
+- Patch
+- Client
+- Device Management
+Agreements:
+- AgreementLabel: Automox Master Services Agreement
+  AgreementUrl: https://www.automox.com/legal/master-services-agreement
+ReleaseNotes: Agent 1.42.22 Release Notes
+ReleaseNotesUrl: https://help.automox.com/hc/en-us/articles/15180475315860-Agent-1-42-22-Release-Notes
```
Comment (stephengillie) [inline manifests/a/Automox/AutomoxAgent/1.42.22/Automox.AutomoxAgent.locale.en-US.yaml:29]:
```suggestion
# ReleaseNotesUrl: 
```

File: manifests/a/Automox/AutomoxAgent/1.42.22/Automox.AutomoxAgent.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,32 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: Automox.AutomoxAgent
+PackageVersion: 1.42.22
+PackageLocale: en-US
+Publisher: Automox
+PublisherUrl: https://www.automox.com/
+PublisherSupportUrl: https://help.automox.com/hc/en-us
```
Comment (stephengillie) [inline manifests/a/Automox/AutomoxAgent/1.42.22/Automox.AutomoxAgent.locale.en-US.yaml:9]:
```suggestion
# PublisherSupportUrl: 
```

**Review by stephengillie — CHANGES_REQUESTED** [review]

**Review by stephengillie — APPROVED** [review]

**Pull request merged**